### PR TITLE
Add docker ps status descriptions

### PR DESCRIPTION
**Pull request by sam-thibault**
- addresses https://github.com/docker/cli/issues/502

**- What I did**

Added a table to docker ps documentation to describe the meaning of each status.

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: docs/reference/commandline/ps.md [hunk 1]
```diff
@@ -174,9 +174,19 @@ Any of these events result in a `137` status:
...
 
-The `status` filter matches containers by status. You can filter using
-`created`, `restarting`, `running`, `removing`, `paused`, `exited` and `dead`. For example,
-to filter for `running` containers:
+The `status` filter matches containers by status. The possible values for the container status are:
+
+| Status                | Description                                                                                                                          |
+|:----------------------|:-------------------------------------------------------------------------------------------------------------------------------------|
+| `created`             | A container that has never been started. |
+| `running`             | A running container, started by either `docker start` or `docker run`. |
```
Comment (thaJeztah) [inline docs/reference/commandline/ps.md:182]: I was curious if the filter would show paused (but running) containers as "running", but it looks like we don't show paused containers when filtering on `running`

(We probably have some wording somewhere though to explain that `paused` and `running` are not mutually exclusive)

File: docs/reference/commandline/ps.md [hunk 2]
```diff
@@ -174,9 +174,19 @@ Any of these events result in a `137` status:
...
 #### status
 
-The `status` filter matches containers by status. You can filter using
-`created`, `restarting`, `running`, `removing`, `paused`, `exited` and `dead`. For example,
-to filter for `running` containers:
+The `status` filter matches containers by status. The possible values for the container status are:
+
+| Status                | Description                                                                                                                          |
+|:----------------------|:-------------------------------------------------------------------------------------------------------------------------------------|
+| `created`             | A container that has never been started. |
```
Comment (thaJeztah) [inline docs/reference/commandline/ps.md:181]: Could you format the table to have the borders aligned? I think we format all our tables like that (and I know for sure my IDE will complain about them not being formatted ... but that's on my machine 😂 )

**Review by thaJeztah — COMMENTED** [review]
overall LGTM, but left a comment about formatting.

**Review by dvdksn — APPROVED** [review]
LGTM, thanks @sam-thibault

File: docs/reference/commandline/ps.md (hunk 2 above)
Comment (sam-thibault) [reply docs/reference/commandline/ps.md]: I tried using Prettier.  Is this the formatting you are after?

File: docs/reference/commandline/ps.md (hunk 1 above)
Comment (sam-thibault) [reply docs/reference/commandline/ps.md]: Ok, I won't worry about adding this right now.

File: docs/reference/commandline/ps.md (hunk 2 above)
Comment (thaJeztah) [reply docs/reference/commandline/ps.md]: Yup

Well, mostly, but I have an open ticket in JetBrain's issue tracker for that 😞 https://youtrack.jetbrains.com/issue/IDEA-332005/Markdown-table-headers-formatting-does-not-match-GFM-spec

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by dvdksn** [comment]
Should we backport or leave it for v25?

**Comment by thaJeztah** [comment]
@dvdksn yes, we should backport this one to the v24 branch; didn't come round to that yet, but feel free to beat me to it :)

**Pull request merged**